src/rebar3_gpb_compiler.erl

```diff
@@ -157,48 +152,47 @@ get_dependencies(Proto, Protos, GpbOpts) ->
     %% go through each of the protos and for each one
     %% check if it included the provided proto, return
     %% the ones that do
-    lists:filtermap(fun(Proto0) ->
-                      filter_included_proto(Proto0, Proto, GpbOpts)
-                    end, Protos).
+    lists:filtermap(fun(Proto0) -> filter_included_proto(Proto0, Proto, GpbOpts) end, Protos).
 
 filter_included_proto(Source, Dep, GpbOpts) ->
     InputsOutputs = gpb_compile:list_io(Source, GpbOpts),
     {sources, SourceAndImports} = lists:keyfind(sources, 1, InputsOutputs),
     Imports = tl(SourceAndImports),
     case lists:member(Dep, Imports) of
-        true -> {true, Source};
-        false -> false
+        true ->
+            {true, Source};
+        false ->
+            false
     end.
 
-
 find_first_match(WantedProto, []) ->
     rebar_api:abort("Filtered proto file not found in path: ~p", [WantedProto]);
-
 find_first_match(WantedProto, [Head | RemainingProtos]) ->
-    case is_wanted_proto(WantedProto, Head)of
+    case is_wanted_proto(WantedProto, Head) of
         true ->
             Head;
         false ->
             find_first_match(WantedProto, RemainingProtos)
     end.
 
-
 is_wanted_proto(WantedProto, ProtoPath) ->
     case string:sub_string(ProtoPath, length(ProtoPath) - length(WantedProto) + 1) of
-        WantedProto -> true;
-        _Other -> false
+        WantedProto ->
+            true;
+        _Other ->
+            false
     end.
```

Comment:
```suggestion
    WantedProto == string:sub_string(ProtoPath, length(ProtoPath) - length(WantedProto) + 1).
```

Reply:
Weirdly enough Github doesn't allow me to commit the suggestion from the UI; I will include this change on the next commit :)